source/jormungandr/jormungandr/street_network/tests/handimap_test.py

```diff
@@ -0,0 +1,421 @@
+    assert language == "en-EN"
+
+
+def get_language_parameter_handimap_func_language_invalid_test():
+    instance = MagicMock()
+    handimap = Handimap(instance=instance, service_url='bob.com', username='aa', password="bb")
+    request = {"_handimap_language": "toto"}
+    language = handimap.get_language_parameter(request)
+    assert language == "fr-FR"
+
+
+def get_language_parameter_handimap_func_language_invalid_test():
+    instance = MagicMock()
+    handimap = Handimap(instance=instance, service_url='bob.com', username='aa', password="bb")
+    request = {"_handimap_language": "english"}
+    language = handimap.get_language_parameter(request)
+    assert language == "en-EN"
+
+
+def make_request_arguments_walking_details_handimap_func_invalid_test():
+    res = Handimap._make_request_arguments_walking_details(1.5, "en-EN")
+    assert res["costing"] == "walking"
+    assert res["costing_options"]["walking"]["walking_speed"] == 5.0
+    assert res["directions_options"]["language"] == "en-EN"
+
+
+def make_request_arguments_direct_path_handimap_func_test():
+    origin = type_pb2.PtObject()
+    origin.embedded_type = type_pb2.POI
+    origin.poi.coord.lon = 42.42
+    origin.poi.coord.lat = 41.41
+
+    destination = type_pb2.PtObject()
+    destination.embedded_type = type_pb2.POI
+    destination.poi.coord.lon = 32.41
+    destination.poi.coord.lat = 31.42
+    arguments_direct_path = Handimap._make_request_arguments_direct_path(origin, destination, 1.5, "en-EN")
+    assert arguments_direct_path["costing"] == "walking"
+    assert arguments_direct_path["directions_options"]["units"] == "kilometers"
+    assert arguments_direct_path["directions_options"]["language"] == "en-EN"
+    assert len(arguments_direct_path["locations"]) == 2
+    assert arguments_direct_path["locations"][0]["lat"] == origin.poi.coord.lat
+    assert arguments_direct_path["locations"][0]["lon"] == origin.poi.coord.lon
+    assert arguments_direct_path["locations"][1]["lat"] == destination.poi.coord.lat
+    assert arguments_direct_path["locations"][1]["lon"] == destination.poi.coord.lon
+
+
+def format_coord_handimap_func_test():
+    pt_object = type_pb2.PtObject()
+    pt_object.embedded_type = type_pb2.POI
+    pt_object.poi.coord.lon = 42.42
+    pt_object.poi.coord.lat = 41.41
+    coords = Handimap._format_coord(pt_object)
+    assert coords["lon"] == pt_object.poi.coord.lon
+    assert coords["lat"] == pt_object.poi.coord.lat
+
+
+def get_response_handimap_test():
+    instance = MagicMock()
+    handimap = Handimap(instance=instance, service_url='bob.com', username='aa', password="bb")
+    resp_json = direct_path_response_valid()
+
+    origin = make_pt_object(type_pb2.ADDRESS, lon=-1.6761174, lat=48.1002462, uri='HandimapStart')
+    destination = make_pt_object(type_pb2.ADDRESS, lon=-1.6740057, lat=48.097592, uri='HandimapEnd')
+    fallback_extremity = PeriodExtremity(str_to_time_stamp('20220503T060000'), True)
```

Comment:
can you test with  `PeriodExtremity(str_to_time_stamp('20220503T060000'), False)` plz